src/ucp/am/eager.inl

```diff
@@ -16,21 +16,22 @@
 
 
 static UCS_F_ALWAYS_INLINE void
-ucp_am_fill_header(ucp_am_hdr_t *hdr, ucp_request_t *req)
+ucp_am_eager_fill_header(ucp_am_hdr_t *hdr, ucp_request_t *req)
 {
     hdr->am_id         = req->send.msg_proto.am.am_id;
     hdr->flags         = req->send.msg_proto.am.flags;
     hdr->header_length = req->send.msg_proto.am.header_length;
 }
 
-static UCS_F_ALWAYS_INLINE size_t ucp_am_send_req_total_size(ucp_request_t *req)
+static UCS_F_ALWAYS_INLINE size_t
+ucp_am_eager_send_req_total_size(ucp_request_t *req)
 {
     return req->send.state.dt_iter.length +
            req->send.msg_proto.am.header_length;
 }
 
 static UCS_F_ALWAYS_INLINE void
-ucp_am_pack_user_header(void *buffer, ucp_request_t *req)
+ucp_am_eager_pack_user_header(void *buffer, ucp_request_t *req)
```

Comment:
can these funcs be used for rndv later? Maybe no need to rename them

Reply:
yes, it will be re-used by rndv. i'll revert this rename.

Reply:
Reverted renaming of `ucp_am_fill_header` and `ucp_am_pack_user_header`.